frontend/src/components/pages/NewAccountPage.tsx

```diff
@@ -1,15 +1,75 @@
 import { Container, Divider, Text } from "@chakra-ui/react";
 import React, { useState } from "react";
+import { gql, useMutation } from "@apollo/client";
 import SignupNavbar from "../common/SignupNavbar";
 import AccountForm from "../user/AccountForm";
 import ProfilePhotoForm from "../user/ProfilePhotoForm";
+import Loading from "../common/Loading";
+import ErrorModal from "../common/ErrorModal";
+
+const CREATE_VOLUNTEER_USER = gql`
+  mutation CreateVolunteerUser($volunteer: CreateVolunteerUserDTO!) {
+    createVolunteerUser(volunteerUser: $volunteer) {
+      id
+    }
+  }
+`;
+
+const CREATE_EMPLOYEE_USER = gql`
+  mutation CreateEmployeeUser($employee: CreateEmployeeUserDTO!) {
+    createEmployeeUser(employeeUser: $employee) {
+      id
+    }
+  }
+`;
 
 const NewAccountPage = (): React.ReactElement => {
   const [profilePhoto, setProfilePhoto] = useState<string>("");
   const [isAdmin] = useState<boolean>(false);
+  const [
+    createEmployee,
+    { loading: createEmployeeLoading, error: createEmployeeError },
+  ] = useMutation(CREATE_EMPLOYEE_USER);
+  const [
+    createVolunteer,
+    { loading: createVolunteerLoading, error: createVolunteerError },
+  ] = useMutation(CREATE_VOLUNTEER_USER);
+
+  if (createEmployeeLoading || createVolunteerLoading) {
+    return <Loading />;
+  }
+  const isError = createEmployeeError || createVolunteerError;
+
+  const onEmployeeCreate = async (employee: any) => {
+    try {
+      await createEmployee({
+        variables: {
+          employee,
+        },
+      });
+    } catch (e) {
+      /* eslint-disable-next-line no-alert */
+      alert("Error: Issue with creating employee user");
+      // TODO: Render error modal instead
+    }
```

Comment:
```suggestion
      await createEmployee({
        variables: {
          employee,
        },
      });
```
actually, I don't think we need to catch errors since we handle them in the error modal